app/controllers/export_controller.rb

```diff
@@ -4,6 +4,10 @@ class ExportController < ApplicationController
   before_action :update_totp, :only => [:finish]
   authorize_resource :class => false
 
+  content_security_policy do |policy|
```

Comment:
Should this be `content_security_policy(:only => :embed) do |policy|`?

Reply:
Yes indeed, now fixed.